### ISSUE TITLE
Check (over) scroll behaviour

### ISSUE THREAD

**Issue by JobDoesburg** [SNiC-foundation/mobilit-singlepage#16]
I have the feeling that the current scroll-snap settings cause some scrolling glitches on certain browsers. For example, on Safari macOS over scrolling does not work (which isn't too bad) but it might indicated some problematic CSS. Also scrolling glitches a bit on some browsers

**Comment by se-bastiaan** [comment]
Is the scroll-snap thingie really necessary for the experience? I am using Firefox on Windows currently and it really feels like my scrollwheel is broken.

**Comment by JobDoesburg** [comment]
> Is the scroll-snap thingie really necessary for the experience? I am using Firefox on Windows currently and it really feels like my scrollwheel is broken.

I'm in doubt. I thought of just removing the scroll snap, but on the other hand this looks bad as well:

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/7915741/107414920-f3df1980-6b12-11eb-8fde-b5c63556f4f5.png">

**Comment by JobDoesburg** [comment]
The biggest problem is illustrated in example 3 on https://css-tricks.com/practical-css-scroll-snapping/. Intuitively, I would have added `scroll-snap-type: y proximity;` to `body` and `header, section` have `scroll-snap-align: start;`. If we do this, scrolling works perfectly as expected and it doesn't feel like the page is fighting back (what @se-bastiaan describes). 

However, this does not work properly, at least on certain browsers. It does not does not snap correctly on the right places (but it snaps somewhere halfway each element, or with some offset). I don't know why this happens, but it is reported on some forums as well. According to the page above, the `scroll-snap-type` property should be added to `html`. Also check out https://stackoverflow.com/questions/53416348/css-scroll-snapping-vertical-not-working and https://stackoverflow.com/questions/54903104/is-it-possible-to-apply-css-scroll-snap-to-the-html-tag-rather-than-the-body-tag

The solution I now implemented does snap correctly, but because we've hardcoded the `body` and `html` to have a height of `100vh`, the scrolling does not work as natively as you would expect from other websites. So that's not nice...

https://blog.logrocket.com/how-to-use-css-scroll-snap/ also tells us to use `-webkit-overflow-scrolling: touch;`
